dan_layer/engine_types/src/hashing.rs

```diff
@@ -29,11 +29,10 @@ use blake2::{
 use digest::Digest;
 use serde::Serialize;
 use tari_bor::encode_into;
-use tari_crypto::{hash_domain, hashing::DomainSeparation};
+use tari_crypto::hashing::DomainSeparation;
+use tari_hash_domains::TariEngineHashDomain;
 use tari_template_lib::Hash;
 
-hash_domain!(TariEngineHashDomain, "com.tari.dan.engine", 0);
```

Comment:
Is there a reason that this DAN-level hash domain goes in the common hash domain crate?

IMO only minotari hash domains that are needed to interact with the base layer should go there. We can manage L2 hash domains in this repo

Reply:
It's because there is a template registration tab in the console
https://github.com/tari-project/tari/blob/development/applications/minotari_console_wallet/src/ui/components/register_template_tab.rs#L541

Reply:
Ahh ok thanks